Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -162,6 +162,16 @@
             "required": false,
             "groupName": "AdditionalDeploymentOptions",
             "helpMarkDown": "Select the option to take the AzureRM Web App offline by placing an app_offline.htm file in the root directory of the Web App before the sync operation begins. The file will be removed after the sync operation completes successfully."
+        },
+        {
+            "name": "RenameFilesFlag",
+            "type": "boolean",
+            "label": "Enable MSDEPLOY_RENAME_FILES_FLAG",
```

Comment:
"Rename Files" should be good enough. The flag details should be in help string.